frontend/amundsen_application/static/js/pages/TableDetailPage/LineageList/index.spec.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import LineageList from './index';
+
+describe('LineageList', () => {
+  let wrapper;
+  let props;
+  beforeAll(() => {
+    props = {
+      items: [
+        {
+          badges: [],
+          cluster: 'gold',
+          database: 'hive',
+          key: 'hive://gold.test_schema/test_table1',
+          level: 0,
+          name: 'test_table1',
+          parent: null,
+          schema: 'test_schema',
+          source: 'hive',
+          usage: 0,
+        },
+      ],
+      direction: 'both',
+    };
+    wrapper = shallow<typeof LineageList>(<LineageList {...props} />);
```

Comment:
Thanks for the tests!

However, this format of tests with 'wrapper' as a let and using 'shallow' is something legacy we want to move out of.

Check this file https://github.com/amundsen-io/amundsen/blob/7d8088429b224958cc212f9aa4d0fca2f3ead655/frontend/amundsen_application/static/js/components/Alert/index.spec.tsx#L11 for a more up to date pattern for our tests.

Reply:
Thanks for pointing out, I've updated the tests. Please check again.